src/js/media-captions-button.js

```diff
@@ -0,0 +1,207 @@
+    if (ccIsOn) {
+      // Closed Captions is on. Clicking should disable any currently showing captions (and subtitles, if relevant)
+      // For why we are requesting tracks to `mode="disabled"` and not `mode="hidden"`, see: https://github.com/muxinc/media-chrome/issues/60
+      const captionsShowingStr = this.getAttribute(
+        MediaUIAttributes.MEDIA_CAPTIONS_SHOWING
+      );
+      // If we have currently showing captions track(s), request for them to be disabled.
+      if (captionsShowingStr) {
+        const evt = new window.CustomEvent(
+          MediaUIEvents.MEDIA_DISABLE_CAPTIONS_REQUEST,
+          { composed: true, bubbles: true, detail: captionsShowingStr }
+        );
+        this.dispatchEvent(evt);
+      }
+      const subtitlesShowingStr = this.getAttribute(
+        MediaUIAttributes.MEDIA_SUBTITLES_SHOWING
+      );
+      // If we have currently showing subtitles track(s) and we're using subtitle fallback (true/"on" by default), request for them to be disabled.
+      if (subtitlesShowingStr && !this.hasAttribute('no-subtitles-fallback')) {
+        const evt = new window.CustomEvent(
+          MediaUIEvents.MEDIA_DISABLE_SUBTITLES_REQUEST,
+          { composed: true, bubbles: true, detail: subtitlesShowingStr }
+        );
+        this.dispatchEvent(evt);
+      }
+    } else {
+      // Closed Captions is off. Clicking should show the first relevant captions track or subtitles track if we're using subtitle fallback (true/"on" by default)
+      const [ccTrackStr] =
+        splitTextTracksStr(
+          this.getAttribute(MediaUIAttributes.MEDIA_CAPTIONS_LIST) ?? ''
+        ) ?? [];
+        if (ccTrackStr) {
+        // If we have at least one captions track, request for the first one to be showing.
+        const evt = new window.CustomEvent(
+          MediaUIEvents.MEDIA_SHOW_CAPTIONS_REQUEST,
+          { composed: true, bubbles: true, detail: ccTrackStr }
+        );
+        this.dispatchEvent(evt);
+
+      } else if (!this.hasAttribute('no-subtitles-fallback')) {
+        // If we don't have a captions track and we're using subtitles fallback (true/"on" by default), check if we have any subtitles available.
+        const [subTrackStr] =
+          splitTextTracksStr(
+            this.getAttribute(MediaUIAttributes.MEDIA_SUBTITLES_LIST) ?? ''
+          ) ?? [];
+        if (subTrackStr) {
+          // If we have at least one subtitles track (and didn't have any captions tracks), request for the first one to be showing as a fallback for captions.
+          const evt = new window.CustomEvent(
+            MediaUIEvents.MEDIA_SHOW_SUBTITLES_REQUEST,
+            { composed: true, bubbles: true, detail: subTrackStr }
+          );
+          this.dispatchEvent(evt);
+        }
+      }
+      else {
+        // If we end up here, it means we have an enabled CC-button that a user has clicked on but there are no captions and no subtitles (or we've disabled subtitles fallback).
```

Comment:
I don't think we need to solve this in this PR, but I'm wondering if this needs to be elevated to the end-user's view. Otherwise they could just be clicking this button and seeing no result. Ideally the UI isn't even showing the CC button if in this case, so we should probably at minimum give an example of how to do that.